docs/source/migration/4.0.mdx

```diff
@@ -61,14 +61,27 @@ client.subscription(MySubscription()).toFlow().collect { response ->
 }
 ```
 
+### Ignoring partial data
+
+With GraphQL it is possible to receive partial data (both `data` and `errors` are filled), but clients sometimes don't want to handle it and prefer to consider any errors as fatal. To help with this, an option `ApolloClient.ignorePartialData` has been added to force `ApolloResponse.data` to be `null` even if it wasn't null in the response, when there are errors. This can simplify error handling at the call site, e.g.:
+
+```kotlin
+val response = apolloClient.query(MyQuery()).execute()
+if (response.data == null) {
+  // There were network or GraphQL error(s)
+} else {
+  // No errors
+}
+```
```

Comment:
This doesn't belong in the migration guide but more in the error handling doc. It's "just" a new feature, nothing breaking.

Reply:
Oops, agreed!